wiki/Installation.md

```diff
@@ -42,8 +42,7 @@ import SwiftCurrent_UIKit
 ```swift
 import SwiftCurrent
 ```
-
-You will need to build your own [Orchestration Responders](https://gitcdn.link/cdn/wwt/SwiftCurrent/faf9273f154954848bf6b6d5c592a7f0740ef53a/docs/Protocols/OrchestrationResponder.html) for your domains.
+`Note:` When using SwiftCurrent for a custom domain, you will need to build the associated [Orchestration Responders](https://gitcdn.link/cdn/wwt/SwiftCurrent/faf9273f154954848bf6b6d5c592a7f0740ef53a/docs/Protocols/OrchestrationResponder.html).
```

Comment:
`you will need to build the associated [Orchestration Responders]` for some reason made me think that there would be an Orchestration Responder I needed to get and then build as well, versus me having to create it.  Perhaps this would read better?

```suggestion
`Note:` When using SwiftCurrent for a custom domain, you will need to build out the associated [Orchestration Responders](https://wwt.github.io/SwiftCurrent/Protocols/OrchestrationResponder.html).
```
I like this one better, thanks @wiemerm !